normalView/normalView.js

```diff
@@ -0,0 +1,34 @@
+let button = document.getElementById("button1");
+let notification="on";
+
+localStorage.getItem("notification",notification);
+
+
+
+//localStorage.setItem("count",count);
+button.onclick= function() { 
+    // var property = document.getElementById("button");
```

Comment:
The id of the button in the html template is 'button1'. The buttons should have a more meaningful name, like in variables or functions, meaningful names will help reading the code and avoiding mistakes. A more meaningful name would be 'pauseButton'

```const pauseButton = document.getElementById("button1");```
```const pauseButton = document.getElementById("pauseButton");```